README_discovery.md

```diff
@@ -108,6 +108,7 @@ mulesoft:
 | MULESOFT_DISCOVERYIGNORETAGS                                       | mulesoft.discoveryIgnoreTags                                        | Comma-separated black list of tags that, if any are present, will prevent an API being publised to Amplify Central. Take precedence over MULESOFT_DISCOVERYTAGS                                                                                                                                          | (empty tag list)                                                     |
 | MULESOFT_DISCOVERYTAGS                                             | mulesoft.discoveryTags                                              | Comma-separated list of tags that, if any are present, will allow an API to be publised to Amplify Central. All APIs are discovered if not tags are specified                                                                                                                                            | (empty tag list)                                                     |
 | MULESOFT_ENVIRONMENT                                               | mulesoft.environment                                                | The Mulesoft Anypoint Exchange the agent connects to, e.g. Sandbox.                                                                                                                                                                                                                                      |                                                                      |
+| MULESOFT_ORGNAME                                                   | mulesoft.orgName                                                    | The Mulesoft Anypoint Business Unit the agent connects to                                                                                                                                                                                                                                                |                                                                      |
```

Comment:
Business group. Also minor one - why not called it something like MULESOFT_BUSINESS_GRP_NAME? Because Org is ambiguous, it could refer to Amplify too

Reply:
Freezing for  demo. Will address as part of helm based changes.